### PR TITLE
Allow store-specific searches & ignore failed requests

### PR DESCRIPTION
**Pull request by Jade-GG**

### CODE REVIEW

**Review by indykoning — APPROVED** [review]

**Comment by royduin** [comment]
Conflicts 😇

**Comment by Jade-GG** [comment]
> Conflicts 😇

Not anymore 😇

File: resources/js/components/Search/Autocomplete.vue [hunk 1]
```diff
@@ -87,11 +92,12 @@ export default {
                     method: 'POST',
                     headers: { 'Content-Type': 'application/json', Authorization: auth },
                     data: JSON.stringify(esQuery),
+                    validateStatus: false,
                 }).then(async (response) => {
                     const responseData = await response.json()
 
                     self.results[name] = responseData?.hits ?? []
-                    self.results.count += self.results[name]?.hits?.length ?? 0
+                    self.results.count += self.results[name]?.hits?.length ??
```
Comment (royduin) [inline resources/js/components/Search/Autocomplete.vue:100]: No fallback?

File: resources/js/components/Search/Autocomplete.vue [hunk 2]
```diff
@@ -87,6 +92,7 @@ export default {
                     method: 'POST',
                     headers: { 'Content-Type': 'application/json', Authorization: auth },
                     data: JSON.stringify(esQuery),
+                    validateStatus: false,
```
Comment (royduin) [inline resources/js/components/Search/Autocomplete.vue:95]: Isn't that an Axios thing?

**Comment by royduin** [comment]
Any updates?

**Comment by royduin** [comment]
Any info?

**Comment by Jade-GG** [comment]
When working with a multistore setup, sometimes some of the stores only require some of the indexes to be searched. For example, if you have a blog on only one of your three multistore sites, you wouldn't want that blog index to also be searched on the other two. Thus, with this change we can choose which indexes get searched per store.

**Comment by royduin** [comment]
And where do we configure that? Do we provide an example and/or some info in the docs?

**Comment by royduin** [comment]
And the docs? Currently there isn't mentioned anything about indexing "additionals"

**Comment by royduin** [comment]
Follow-up with internal reference: RAP-1054

**Pull request merged**